packages/app/src/specs/SpecsList.cy.tsx

```diff
@@ -164,6 +164,25 @@ describe('<SpecsList />', { keystrokeDelay: 0 }, () => {
           })
         })
       })
+
+      it('displays the list as expected visually at various widths', () => {
+        cy.get('[data-cy="spec-list-file"]')
+        .should('have.length.above', 2)
+        .should('have.length.below', specs.length)
+
+        cy.wait(100) // there's an intentional 50ms delay in the code, lets just wait it out
+
+        cy.viewport(500, 850)
+        cy.percySnapshot('narrowest')
+        cy.viewport(650, 850)
+        cy.percySnapshot('narrow')
+        cy.viewport(800, 850)
+        cy.percySnapshot('medium')
+        cy.viewport(1200, 850)
+        cy.percySnapshot('wide')
+        cy.viewport(2000, 850)
+        cy.percySnapshot('widest')
```

Comment:
Are we using the [Tailwind CSS viewport breakpoints](https://tailwindcss.com/docs/responsive-design)? If so, it would be nice to mirror the same values

Reply:
I believe columns are using those breakpoints for changing titles and showing/hiding, but the truncating behavior kinda falls in between, depending on how wide the column is, so I think I'd rather leave this as is it.

Though a more general page level percy helper command could be really useful so we cover more breakpoints without having to be too explicit about it 🤔

Reply:
A `cy.percyResponsive` that did the viewports and snapshots for you.